IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>For this plot, and the next one, when I ran them, the axis labels were cut off a little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>which browser have you used? I remember the axis labels issue in Safari. </p>